CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 * Upgrade dependency `org.apache.httpcomponents:httpclient` to v4.5.8 to patch vulnerabilities.
 
 ### Fixed
-* Fixed an issue where DataAPI's `requestRecursive` method would throw an exception upon receiving a response from Data API endpoints that set the "data" field of the response to an object (like the `itembank/questions` endpoint when `item_references` is included in the request).
+* Fixed an issue where Data API's `requestRecursive` method would throw an exception upon receiving a response from Data API endpoints that set the "data" field of the response to an object (like the `itembank/questions` endpoint when `item_references` is included in the request).
```

Comment:
I wonder if we need `DataAPI` here, as I thing @flakeparadigm was refering to the actual Java class.

Reply:
Yeah, Edwin and I are already discussing that.

Reply:
(Though my original capitalization was also incorrect 😅 haha)